packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -74,29 +80,45 @@ const ModalContent = React.forwardRef((props, ref) => {
     role,
     isFullscreen,
     isScrollable,
+    rootSelector,
     zIndex,
     ...customProps
   } = props;
-  let zIndexLayer = '6000';
 
+  useEffect(() => {
+    // Store element that was last focused prior to modal opening
+    const modalTrigger = document.activeElement;
+    showModalDomUpdates(ref.current, rootSelector);
```

Comment:
Using the forwarded ref here threw me for a loop. I think this will work, but it's a new one for me.